style.css

```diff
@@ -611,3 +622,837 @@ body {
   list-style: none;
   padding-left: 80px;
 }
+
+@media screen and (min-width: 768px) {
+  * {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  #toolbar {
+    width: 100%;
+    height: 72px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    background-color: #3c3c3c;
+  }
+
+  #heading {
+    width: 20%;
+    display: flex;
+    align-items: center;
```

Comment:
- [ ] Kindly make sure your page is not overlapping on the right on the movie version. The content must fit all the available width of the actual page.
![image](https://user-images.githubusercontent.com/97100091/193300170-52d375e2-304d-4c02-b00c-7e3bd2213d99.png)

Reply:
@christianbiring1 I do not understand this issue can you please elaborate more ?